tests/browser/features/insufficient.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Happy path
+
+  Successful journey through the system and back to the RP
```

Comment:
Is this description correct for this test file?

Reply:
Updated the description to be more correct